hecuba_core/src/py_interface/UnitParser.cpp

```diff
@@ -206,6 +206,76 @@ PyObject *TextParser::c_to_py(const void *payload) const {
     return PyUnicode_FromString(d);
 }
 
+/***Date parser ***/
+
+DateParser::DateParser(const ColumnMeta &CM) : UnitParser(CM) {
+    if (CM.size != sizeof(int64_t *))
+        throw ModuleException("Bad size allocated for a date");
+    if (!PyDateTimeAPI) PyDateTime_IMPORT;
+}
+
+int16_t DateParser::py_to_c(PyObject *obj, void *payload) const {
+    PyDateTime_IMPORT;
+    if (obj == Py_None) return -1;
+    if (PyDate_CheckExact(obj)) {
+        time_t time_now;
+        struct tm *timeinfo;
+        time(&time_now);
```

Comment:
why time(time_now) ?

Reply:
I think you can do the same by declaring:
```c++
struct tm timeinfo;
timeinfo->tm_year = PyDateTime_GET_YEAR(obj) - 1900;
// and so on
```
and forget about the "time_now"

Reply:
The implementation requires all the previous definitions in order to get the correct tm pointer for calling mktime